multiaccounts/settings/columns.go

```diff
@@ -452,7 +452,11 @@ var (
 		dBColumnName:   "omit_transfers_history_scan",
 		valueHandler:   BoolHandler,
 	}
-
+	MnemonicWasNotShown = SettingField{
```

Comment:
`mnemonic` sure we do, because `MnemonicWasNotShown` should be set to false even if user has not finished backing up flow. So that if it had appeared on the screen it is wasted

Reply:
Yes, makes sense.